src/commands/push-with-dynamic-tag.yml

```diff
@@ -20,5 +20,5 @@ steps:
   - deploy:
       command: >
         docker push <<parameters.registry>>/<<
-        parameters.image>>:${<<parameters.tag-env>>}
+        parameters.image>> --all-tags
```

Comment:
This might be problematic. There are projects that push tags individually so this might mean we push twice.  
Can we test this with `[source-liveramp-rmn](https://github.com/GoodwayGroup/source-liveramp-rmn)`

Reply:
This has been tested [here](https://app.circleci.com/pipelines/github/GoodwayGroup/source-liveramp-rmn/383/workflows/5b4dbf25-6f9e-49e5-a561-fbd8800d8ff0/jobs/695). It is working as expected.